SearchService2/solr-management/ensure_solr_running.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+SOLR_EXTRACT_DIR=/opt/
+SOLR_VAR_DIR=/var/solr/
+SOLR_MANAGEMENT_DIR="$( realpath --strip "$( dirname "${BASH_SOURCE[0]}" )" )"
+
+# Ensure Solr is installed
+VERSION=7.6.0 "${SOLR_MANAGEMENT_DIR}/ensure_solr_installed.sh"
```

Comment:
Since a lack of the installation during runtime would indicate that something went wrong during the build, it might make more sense to fail rather than try to re-install.